website/blog/2025/08/08-13-keeping-track-of-your-resources-with-inventory.md

```diff
@@ -0,0 +1,539 @@
+---
+title: Keeping track of your cloud resources with Gardener Inventory
+linkTitle: Keeping track of your cloud resources with Gardener Inventory
+newsSubtitle: Aug 13, 2025
+publishdate: 2025-08-13
+authors:
+- avatar: https://avatars.githubusercontent.com/dnaeon
+  email: dnaeon@gmail.com
+  login: dnaeon
+  name: Marin Atanasov Nikolov
+aliases: [/blog/2025/08/13/getting-started-with-gardener-inventory]
+---
+Running Kubernetes clusters at scale comes with it's own challenges.
+
+How do you keep track of the resources under your control? How can you tell if a
+resource has become orphan and start adding up to your monthly cloud cost?
+
+This post introduces [gardener/inventory](https://github.com/gardener/inventory)
+and shows how the Gardener project keeps track of the cloud resources it manages
+in the various cloud providers such as GCP, AWS, Azure and OpenStack.
+
+It will also show how we can identify orphan resources through a simple, yet
+effective mechanism by collecting and reporting from the respective cloud
+providers.
```

Comment:
```suggestion
It will also show how we can identify orphaned resources through a simple, yet
effective mechanism, by collecting and reporting from the respective cloud
providers.
```